airflow/contrib/operators/postgres_to_gcs_operator.py

```diff
@@ -155,6 +156,9 @@ def _write_local_data_files(self, cursor):
                 file_no += 1
                 tmp_file_handle = NamedTemporaryFile(delete=True)
                 tmp_file_handles[self.filename.format(file_no)] = tmp_file_handle
+            row_no += 1
+
+        self.log.info('Received %s rows over %s files', row_no, file_no + 1)
```

Comment:
Shouldn't it be just `file_no` and not `file_no + 1`?

Reply:
Line156 already increments the `file_no` variable.

Reply:
The `file_no` indicates how many files are being written, to avoid that files grow too big. The `row_no` should indicate how many lines are being written. So I understand the confusion. The thing is, if no files are being rolled, `file_no` will still be 0. But if no rows are being written it will say, `written 0 rows over 1 file`. Let me make this more clear.